web/styles/left_sidebar.css

```diff
@@ -1142,12 +1134,10 @@ li.top_left_scheduled_messages {
 }
 
 .topic_search_section {
-    grid-template-columns:
-        var(--left-sidebar-toggle-width-offset) 0 0
-        [filter-box-start] minmax(0, 1fr)
-        minmax(0, max-content)
-        [clear-button-start] var(--left-sidebar-vdots-width)
-        [clear-button-end filter-box-end] 0;
+    padding: 8px 10px;
+    display: grid;
```

Comment:
Okay. So we don't want to do this. The deleted `grid-template-columns` above are so we can keep all the sidebar rows sharing roughly the same grid areas. You should just be able to reference the `clear-button` area if you need to more explicitly place the clear icon, and there should be no need to have to set padding here, either.

Reply:
Thanks for the review and fix Karl!